### DISCUSSION TITLE
List entity properties

### DISCUSSION THREAD

**Discussion by laiyuanliu** [schemaorg/schemaorg#3038] [Q&A]
Hello, I'm relatively new to the RDF world. 

I successfully loaded schema.org ttl file to protege,  is there anyway i can see all of the properties (data property and object property) for an entity? ideally i want to click on a class and see its properties. I tried it with WebProtege and TopBraid Composer, couldn't figure out a way to do it. 

thanks in advance

**Comment by thadguidry** [comment]
Ask the Protégé mailing list
